test/dotnet.Tests/RuntimeConfigTests.cs

```diff
@@ -0,0 +1,210 @@
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Common;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.Tests
+{
+    public class RuntimeConfigTests : TestBase
+    {
+        [Fact]
+        void ParseBasicRuntimeConfig()
+        {
+            var tempPath = GetTempPath();
+            File.WriteAllText(tempPath, Basic);
+            var runtimeConfig = new RuntimeConfig(tempPath);
+            Asset(runtimeConfig);
+        }
+
+        [Fact]
+        void ParseRuntimeConfigWithTrailingCommaShouldThrow()
```

Comment:
Do you expect this behavior forever, or do you plan to loosen up to the earlier behavior after it's possible? If the latter I'd file an issue and put the bug number in a comment here.

Reply:
I think if there is no big cry in preview 4. We should keep this behavior. Since this is the default of system.text.json and may other json files in larger eco system will have the same.